### PR TITLE
Implement @GenIgnore(GenIgnore.PERMITTED_TYPE)

### LINKED ISSUE #104: deployVerticleAwait does not have all method overloads
`deployVerticleAwait()` has only the `(String)` and `(String, DeploymentOptions)` overloads, while the original method has 12 overloads.

Is this need intended?

### PR DESCRIPTION
**Pull request by okou19900722**
Signed-off-by: okou19900722 <ilayan19900722@gmail.com>

 fix #104

### CODE REVIEW

**Comment by okou19900722** [comment]
@gmariotti @vietj PTAL

**Comment by gmariotti** [comment]
Hey @okou19900722, any update on this PR? It would be nice to be able to backport it to 3.6 before the next release

**Comment by okou19900722** [comment]
it's done. and the backported is ready. #114

File: vertx-lang-kotlin-gen/src/main/java/io/vertx/lang/kotlin/KotlinCoroutineGenerator.java [hunk 1]
```diff
@@ -43,9 +43,8 @@ public String render(ClassModel model, int index, int size, Map<String, Object>
     StringWriter buffer = new StringWriter();
     CodeWriter writer = new CodeWriter(buffer);
     ClassTypeInfo type = model.getType();
-    Map<Boolean, List<MethodInfo>> methodGroupMap = model
-      .getMethods()
-      .stream()
+    Stream<MethodInfo> methodStream = Stream.concat(model.getMethods().stream(), model.getAnyJavaTypeMethods().stream());
+    Map<Boolean, List<MethodInfo>> methodGroupMap = methodStream
```
[the remark this answers is not in the document]
Comment (okou19900722) [reply vertx-lang-kotlin-gen/src/main/java/io/vertx/lang/kotlin/KotlinCoroutineGenerator.java]: i think it's clear to read

[the remark this answers is not in the document]
Comment (okou19900722) [reply vertx-lang-kotlin-gen/src/main/java/io/vertx/lang/kotlin/KotlinCoroutineGenerator.java]: @gmariotti i think it's should clear to read. a new commit make it clear

**Comment by gmariotti** [comment]
Thanks again for your contribution @okou19900722

**Pull request merged**